src/main/resources/templates/email/programme-created/v1.0.0.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org" xmlns="http://www.w3.org/1999/html">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+</head>
+<body>
+<h1>Email Message</h1>
+<h2>Subject</h2>
+<th:block th:fragment="subject">NHS England - <span
+    th:text="${not #strings.isEmpty(localOfficeName)} ? ${localOfficeName} : _" th:remove="tag"></span
+  > - Onboarding to your Training programme</th:block>
+<h2>Content</h2>
+<th:block th:fragment="content">
+  <div th:replace="fragments/greeting/v1.0.0 :: body"></div>
+  <p>Congratulations on your appointment to the <b><span
+      th:text="${not #strings.isEmpty(programmeName)} ? ${programmeName} : 'programme'"></span></b>.</p>
```

Comment:
Should "programme" always be included instead of part of the conditional?
e.g.
> Congratulations on your appointment to the programme.
> OR
> Congratulations on your appointment to the **General Practice** programme.

rather than 
> Congratulations on your appointment to the **programme**.
> OR
> Congratulations on your appointment to the **General Practice**.

Reply:
Yeah, it reads quite weird right now, e.g. `Congratulations on your appointment to the Cardiology.`